### ISSUE TITLE
What is the best solution to calculate the content length?

### ISSUE THREAD

**Issue by LeMoussel** [cefsharp/CefSharp#1375]
I want to get the size of the loaded page.
To do, I think to use `IRequestHandler` with `OnResourceResponse` method  like that
````C#
class CEFRequestHandler : IRequestHandler
{
   int pageSize = 0;

   bool IRequestHandler.OnResourceResponse(IWebBrowser browserControl, IBrowser browser, IFrame frame, IRequest request, IResponse response)
   {
            int responseLength = 0;
            Int32.TryParse(response.ResponseHeaders["Content-Length"], out responseLength);
            pageSize += responseLength;

            return false;
   }
} 
````
Is it the best way?

**Comment by amaitland** [comment]
I'm pretty sure `Content-Length` is an optional header, so it's not going to be totally reliable. I'm not aware of any part of the `CEF API` that exposes the request size unfortunately.

**Comment by amaitland** [comment]
Added `IRequestHandler.OnResourceLoadComplete`, which exposes a `receivedContentLength` property.

https://github.com/cefsharp/CefSharp/commit/77824905132e8b7877926f8ec190c421fa3711b8#diff-c9bb088b59252635917e1f1f94830a8aR184

**Comment by amaitland** [comment]
This feature will be available in the next release (`47.0.0`)
